### ISSUE TITLE
Related hyperparameters for FC100 training

### ISSUE THREAD

**Issue by pengshy** [zhangdoudou123/SemFew#2]
Thank you for your code. This is an outstanding job. But I cannot reproduce the results of paper FC100 based on the parameters of training two ImageNet datasets. Could you please provide me with the parameter settings for training FC100.

**Comment by zhangdoudou123** [comment]
The experimental results are related to the training of the backbone network, with no other particular hyper-parameters.

**Comment by pengshy** [comment]
Thanks for your reply. I follow your steps but the accuracy of FC100 is still about 45%. How should I solve it？

**Comment by zhangdoudou123** [comment]
You could change the backbone, for example, resnet or transformer.

**Comment by DruryXu** [comment]
Please try to reproduce the results based on weights we uploaded today, sorry for any inconvenience caused and feel free to contact us if there are any new issues.

**Comment by pengshy** [comment]
Thank you so much!!!!
